mappings/net/minecraft/advancement/Advancement.mapping

```diff
@@ -70,7 +72,9 @@ CLASS net/minecraft/class_161 net/minecraft/advancement/Advancement
 			ARG 8 hidden
 		METHOD method_698 toJson ()Lcom/google/gson/JsonObject;
 		METHOD method_699 toPacket (Lnet/minecraft/class_2540;)V
+			ARG 1 buf
 		METHOD method_700 findParent (Ljava/util/function/Function;)Z
```

Comment:
Maybe this should just be `getParent`?

Reply:
it is trying to find the parent and returns a boolean indicating if the parent is found.